docs/source/nlp/joint_intent_slot.rst

```diff
@@ -3,10 +3,253 @@
+  [NeMo I 2021-01-28 14:52:19 exp_manager:186] Experiments will be logged at results
+  ...
+    label                                                precision    recall       f1           support
+    weather.weather (label_id: 0)                            0.00       0.00       0.00        128
+    weather.temperature (label_id: 1)                        0.00       0.00       0.00          0
+    weather.temperature_yes_no (label_id: 2)                 0.00       0.00       0.00          0
+    weather.rainfall (label_id: 3)                           0.00       0.00       0.00          0
+    weather.rainfall_yes_no (label_id: 4)                    0.00       0.00       0.00          0
+    weather.snow (label_id: 5)                               0.00       0.00       0.00          0
+    weather.snow_yes_no (label_id: 6)                        0.00       0.00       0.00          0
+    weather.humidity (label_id: 7)                           0.00       0.00       0.00          0
+    weather.humidity_yes_no (label_id: 8)                    0.00       0.00       0.00          0
+    weather.windspeed (label_id: 9)                          0.00       0.00       0.00          0
+    weather.sunny (label_id: 10)                             0.00       0.00       0.00          0
+    weather.cloudy (label_id: 11)                            0.00       0.00       0.00          0
+    weather.alert (label_id: 12)                             0.00       0.00       0.00          0
+    context.weather (label_id: 13)                           0.00       0.00       0.00          0
+    context.continue (label_id: 14)                          0.00       0.00       0.00          0
+    context.navigation (label_id: 15)                        0.00       0.00       0.00          0
+    context.rating (label_id: 16)                            0.00       0.00       0.00          0
+    context.distance (label_id: 17)                          0.00       0.00       0.00          0
+    -------------------
+    micro avg                                                0.00       0.00       0.00        128
+    macro avg                                                0.00       0.00       0.00        128
+    weighted avg                                             0.00       0.00       0.00        128
+
+
+Model Evaluation and Inference
+------------------------------
+
+There is no separate script for the evaluation and inference of this model in NeMo, but inside of the example file:
+`examples/nlp/intent_slot_classification/intent_slot_classification.py`
+after the training part is finished you can see the code that evaluates the trained model
+on an evaluation test set and then an example of doing inference using a list of given queries.
+
+For the deployment in the production environment please refer to Jarvis and TLT documentation.
```

Comment:
@vladgets the links are missing

@ericharper @okuchaiev I don't see that we mention Jarvis and/or TLT in the intro.rst? Any plans on adding this?

Reply:
I do not know the correct links

Reply:
Ok, I added some links to Jarvis and TLT docs